source/review.py

```diff
@@ -28,18 +28,22 @@ def getObjectPosition(obj):
 	@return: the TextInfo instance and the Scriptable object the TextInfo instance is referencing, or None on error.
 	@rtype: (L{TextInfo},L{ScriptableObject})
 	"""
-	try:
-		pos = obj.makeTextInfo(textInfos.POSITION_CARET)
-	except (NotImplementedError, RuntimeError):
-		# No caret supported, try first position instead
+	isNavigable: bool = obj._hasNavigableText
```

Comment:
Can you update this function to add type hints and remove them from the docstring?